src/screens/Wishlist/index.jsx

```diff
@@ -1,30 +1,40 @@
 import React from "react";
 import PageHeroHeader from "../../components/PageHeroHeader";
 import ProductCard from "../../components/ProductCard";
+import { useWishlist } from "../../Contexts/WishlistContext";
+import {useNavigate} from 'react-router-dom'
 import "./wishlist.css";
+import Button from "../../components/Header/Button";
 const Wishlist = () => {
-  const imageUrl = "https://picsum.photos/200/300";
+  const { wishlistProducts, isWishlistLoading, wishlistError } = useWishlist();
+  const navigate = useNavigate();
   return (
-
-    <div >
-      <PageHeroHeader title={'Your wishlist'} />
-      <section className="wishlist-container margin-top-60">
-        {[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15].map(
-          (slide, index) => {
-            return (
-              <ProductCard
-                variant="horizontal"
-                imagePath={imageUrl}
-                badgeText={"New Arrival"}
-                title={"One ear 100 headphones black"}
-                price={1890}
-                actualPrice={2099}
-                discountPercentage={10}
-              />
-            );
-          }
+    <div>
+      {/* <PageHeroHeader title={"Your wishlist"} /> */}
+      {isWishlistLoading && <h1 className="margin-top-60">Loading...</h1>}
+        {wishlistError.errorExists && (
+          <h1 className="invalid-field-color margin-top-60">
+            {wishlistError.errorMessage}
+          </h1>
         )}
-      </section>
+      {wishlistProducts.length > 0 ?
+      <section className="wishlist-container margin-top-60">
+        
+        {wishlistProducts.map((product, index) => {
+          return (
+            <ProductCard
+              variant="horizontal"
+              cartActionBtnContainer={false}
+              product={product}
+              key={product._id}
+            />
+          );
+        })}
+      </section> : 
+      <div className="flex-column flex-align-item-center flex-justify-content-center margin-top-60">
+      <h2>Your cart is empty!</h2>
```

Comment:
It should be "Your wishlist is empty" (instead of cart)

Reply:
fixed